wodles/aws/aws_s3.py

```diff
@@ -794,6 +795,12 @@ def iter_files_in_bucket(self, aws_account_id=None, aws_region=None):
             print("ERROR: Unexpected error querying/working with objects in S3: {}".format(err))
             sys.exit(7)
 
+    def check_empty_bucket(self):
+        # exit if bucket is empty
+        if not 'CommonPrefixes' in self.client.list_objects_v2(Bucket=self.bucket, Prefix=self.prefix, Delimiter='/'):
+            print("ERROR: Empty bucket")
```

Comment:
We should change this error message. Having an empty bucket is not a severe error. We only need to notify user that no logs can be found under that bucket without generating too much alarm. Maybe something like 'The bucket {self.bucket} is empty. No logs will be processed.', avoiding the word ERROR in capitals.

Reply:
We cannot avoid `ERROR: ` word because this is needed by `modulesd` in order to format the wodle output, but I am going to change the message as you said.

With `ERROR: `:
```bash
2019/07/26 10:00:52 wazuh-modulesd:aws-s3[27272] wm_aws.c:425 at wm_aws_run_s3(): WARNING: Bucket:  -  Returned exit code 14
2019/07/26 10:00:52 wazuh-modulesd:aws-s3[27272] wm_aws.c:444 at wm_aws_run_s3(): WARNING: Bucket:  -  No files were found in 'wazuh-aws-wodle/macieie/'. No logs will be processed.
2019/07/26 10:00:52 wazuh-modulesd:aws-s3[27272] wm_aws.c:449 at wm_aws_run_s3(): DEBUG: Bucket:  -  OUTPUT: DEBUG: +++ Debug mode on - Level: 1

```

Without `ERROR `:
```bash
2019/07/26 10:08:02 wazuh-modulesd:aws-s3[30685] wm_aws.c:425 at wm_aws_run_s3(): WARNING: Bucket:  -  Returned exit code 14
2019/07/26 10:08:02 wazuh-modulesd:aws-s3[30685] wm_aws.c:446 at wm_aws_run_s3(): WARNING: Bucket:  -  DEBUG: +++ Debug mode on - Level: 1
No files were found in 'wazuh-aws-wodle/macieie/'. No logs will be processed.
```